### ISSUE TITLE
Rejestracja w Sandbox

### ISSUE THREAD

**Issue by tomek2x1** [allegro/allegro-api#10302]
Dzień dobry,
kilka razy próbowałem założyć konto firmowe na SANDBOX na różne dane, ale nie otrzymuję maila do aktywacji konta. Maile jakie wpisywałem przy próbie rejestracji to: it@emultimax.pl, t.lewandowski@emultimax.pl. Proszę o pomoc przy aktywacji konta.

Pozdrawiam,
Tomasz Lewandowski

**Comment by AureliuszBrussy** [comment]
Możesz zalogować się już na konto t.lewandowski@emultimax.pl. Skorzystaj [z tego linku](https://allegro.pl.allegrosandbox.pl/rejestracja/konto-firmowe/wybierz-sciezke?showOnboardingProgressMsg=true) aby przejść proces aktywacji konta.

Jeżeli chodzi o it@emultimax.pl - próba rejestracji jest już przedawniona. Spróbuj jeszcze raz. Upewnij się także, czy na pewno wiadomości z Sandboxa nie wpadły do spamu.

**Comment by tomek2x1** [comment]
Dzięki za pomoc w aktywacji. Czy mógłbym jeszcze prosić o zaakceptowanie dokumentu tożsamości oraz przelewu.

**Comment by AureliuszBrussy** [comment]
@tomek2x1 aktywowałem Twoje konto.

**Comment by tomek2x1** [comment]
Prośba o akceptację oferty

Oferta jest przygotowana
Teraz daj nam czas na jej sprawdzenie.
Gdy ją zatwierdzimy, będzie opublikowana pod linkiem: https://allegro.pl.allegrosandbox.pl/oferta/7776081458

**Comment by MaciejFrackowiak** [comment]
@tomek2x1 Jak widzę oferta jest już dostępna.

**Comment by tomek2x1** [comment]
Tak, dzięki. Proszę jeszcze o aktywację konta użytkownika t.lewandowski@warmtec.pl . Nie wiem o co chodzi ale maile do mnie nie dochodzą. Potrzebuję 2 kont aby przetestować dodawanie listów przewozowych przez API po dokonaniu sprzedaży.

**Comment by MaciejFrackowiak** [comment]
@tomek2x1 Nie widzę takiego konta w naszej bazie. Czy potwierdziłeś rejestrację tego konta? Bez tego nie możemy nic zrobić.

**Comment by oorobak** [comment]
Dzień dobry, podłączę się pod wątek bo dotyczy dokładnie tego samego. Nie przychodzi mi email aktywacyjny konta firmowego z sandboxa na adres allegro@vikingstea.com

**Comment by Lukasz-Zurek** [comment]
@oorobak Wiadomość dotycząca rejestracji konta na sandboxie wysłana została dzisiaj o godzinie 11:52:25.390. Sprawdź proszę folder ze spamem oraz filtry antyspamowe - tego typu wiadomości na naszym środowisku testowym, mogą być traktowane jako spam.

**Comment by oorobak** [comment]
A mogę prosić o ponowienie tego maila? Bo nie mam nic w spamie. Zmieniłem ustawienie filtrów antyspamowych żeby nie usuwał za dużo.

**Comment by oorobak** [comment]
Bo uzupełniając co się wydarzyło. Chodząc po stronie przekierowało mnie na produkcje allegro.pl i tam przez pomyłkę też założyłem konto. I te maile przyszły. Jeden m.in. właśnie o 11:52

**Comment by Lukasz-Zurek** [comment]
Spróbuj ponownie się zarejestrować i potwierdzić rejestrację (najlepiej kopiując link do przeglądarki). Z tego co widzę na formularzu - powyższy adres przechodzi prawidłowo.

**Comment by Lukasz-Zurek** [comment]
@oorobak Extra, że się wyjaśniło. Fakt - wszystkie powiadomienia na sandboxie posiadają załącznik i to może być odbijane przez niektóre skrzynki.
